lib/server.go

```diff
@@ -2320,9 +2339,9 @@ func (srv *Server) _handlePeerMessages(serverMessage *ServerMessage) {
 	case *MsgDeSoInv:
 		srv._handleInv(serverMessage.Peer, msg)
 	case *MsgDeSoVersion:
-		srv.handshakeController._handleVersionMessage(serverMessage.Peer, serverMessage.Msg)
+		srv.handshakeController.handleVersionMessage(serverMessage.Peer, serverMessage.Msg)
```

Comment:
Route the message through NetworkManager so that server doesn't know about the inner workings of NetworkManager
```suggestion
		srv.networkManager.HandleVersionMessage(serverMessage.Peer, serverMessage.Msg)
```